lib/httpotion.ex

```diff
@@ -57,6 +57,8 @@ defmodule HTTPotion.Base do
 
       def process_request_headers(headers, _body), do: process_request_headers(headers)
 
+      def process_request_headers(headers, _body, _options), do: process_request_headers(headers)
```

Comment:
Call with `(headers, body)` here — overriding the `/2` version should still work

Reply:
Fixed in https://github.com/myfreeweb/httpotion/pull/130/commits/d624f496f0218e1f94f4787d3421b907ac3edd9c. I'll squash commits after approval.

Reply:
no need to, I'll squash using github